docs/server/installation.rst

```diff
@@ -393,6 +224,27 @@ And the server will start listening on port 8000. This can be accessed from
 your web browser at `localhost:8000 <http://localhost:8000/>`_.
 
 
+
+.. _installation#next-steps:
+
+Next steps
```

Comment:
By reading *Next steps* I understand the installation document is over. The rest of the sections are probably better off as separate documents, linked from here.